pkgs/by-name/po/poliedros/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  python3Packages,
```

```suggestion
  buildPythonApplication,
```
iirc aliases has been disabled

---

Function called without required argument "buildPythonApplication"

---

it seems like all the other packages in by-name use `python3Packages.buildPythonApplication`

---

it seems like I was wrong yeah, I thought it would have the same behavior as in development/python-modules I'm sorry for causing all this trouble